Packs/IOCParser/Integrations/IOCParser/IOCParser.py

```diff
@@ -0,0 +1,417 @@
+                if not limited_response.get(ioc_type):
+                    limited_response[ioc_type] = []
+                limited_response.get(ioc_type).append(ioc)
+                limit -= 1
+    return limited_response
+
+
+def process_response(response: Dict[str, Any], keys: List[str], limit: int) -> Dict[str, List]:
+    """
+
+
+    :type response: ``Dict[str, Any]``
+    :param response: The data key from the API's response
+
+    :type keys: ``List[str]``
+    :param keys: IOC Types to return
+
+    :type limit: ``int``
+    :param limit: maximum number of results to return
+
+    :return:
+    :rtype: Dict[str, List]
+    """
+
+    try:
+        response_data = response.get('data')
+    except Exception:
+        raise Exception('The response from the API is empty')
+
+    remove_unwanted_keys(response_data, keys)
+    if limit is not None:
+        response_data = limit_response(response_data, int(limit))
+    if not response_data:
+        raise Exception('The response from the API is empty from limit')
+    return response_data
+
+
+def unite_all_tweets_into_dict(twitter_response: Dict[str, Any]) -> None:
+    # The data for this response is a list of "responses", for each tweet of the user
+    """
+    Unites all data from every tweet to a single dictionary
+
+    :type twitter_response: ``Dict[str, List]``
+    :param twitter_response: The data key from the API's response
+    """
+
+    try:
+        response_data = twitter_response.get('data')
+    except Exception:
+        raise Exception('The response from the API is empty')
+
+    united_data = {}
+    for tweet in response_data:
+        for ioc_type, iocs in tweet.get('data').items():
+            for ioc in iocs:
+                if not united_data.get(ioc_type):
+                    united_data[ioc_type] = []
+                united_data.get(ioc_type).append(ioc)
+
+    twitter_response['data'] = united_data
+
+
+''' COMMAND FUNCTIONS '''
+
+
+def test_module(client: Client) -> str:
+    """
+    :type client: ``Client``
+    :param client: IOCParser client to use
+
+
+    :return: 'ok' if test passed, anything else will fail the test.
+    :rtype: ``str``
+    """
+
+    response = client.ioc_from_url('https://pastebin.com/iMzrRXbJ')
+    if (response.get('status') == 'fail') \
+            or (response.get('status') == 'error') \
+            or (response.get('status') is None):
+        return 'Failed to connect with the API'
+    return 'ok'
+
+
+def ioc_from_url_command(client: Client, args: Dict[str, Any]) -> List[CommandResults]:
+    """
+    Returns the results of the Parse IOCs from URL API call
+    Args:
+        client: IOCParser client to use
+        args: All command arguments, ulr, limit and keys (if specified)
+
+    Returns:
+        CommandResults object containing the results of the parse from url as
+        returned from the API and its readable output
+
+    """
+    url = args.get('url')
+    keys = argToList(args.get('keys'))
```

Comment:
```suggestion
    keys = argToList(args.get('keys')) or KEYS
```
Test this behavior yourself to understand it.